lib/rust/parser/src/syntax/tree/builder/src/lib.rs

```diff
@@ -15,6 +15,7 @@
 #![warn(trivial_numeric_casts)]
 #![warn(unused_import_braces)]
 #![warn(unused_qualifications)]
+#![feature(proc_macro_span)]
```

Comment:
This feature breaks IntelliJ understanding of the macro's outputs.

The workaround I used in `profiler` is to set a cfg value from an environment variable in the [build script](https://github.com/enso-org/enso/blob/develop/lib/rust/profiler/macros/build.rs), and then gate the functionality like `#![cfg_attr(enso_enable = "proc_macro_span", feature(proc_macro_span))]` (and at all use sites). We could extract that to a safe-span wrapper for this crate and `profiler` to use.

Reply:
Oh, they must have implemented it! They never responded to my bug report. We can probably remove the workaround from `profiler` then...

Reply:
I just looked into this some more. `proc_macro_span` gates multiple functions; the functionality used by `syntax-tree-builder` works, but the function needed by `profiler` still breaks CLion's understanding of the annotated functions; so it's fine here but `profiler` still needs the workaround.